modules/templates/mailer.go

```diff
@@ -14,6 +15,8 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
+var mailSubjectSplit = regexp.MustCompile(`(?m)^-{3,}\s*$`)
```

Comment:
```suggestion
var mailSubjectSplit = regexp.MustCompile(`(?m)^-{3,}[\s]*$`)
```
why was it changed?

Reply:
Auto simplified by my IDE.

`\s` just equals to `[\s]`, and my IDE doesn't show a warning for it.